### PR TITLE
[AzureLoadTesting] Updated README with examples for management plane

### PR DESCRIPTION
**Pull request by venkatr21**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/loadtestservice/azure-resourcemanager-loadtestservice/pom.xml [hunk 1]
```diff
@@ -7,6 +7,43 @@
     <relativePath>../../parents/azure-client-sdk-parent</relativePath>
   </parent>
 
+  <build>
```
[the remark this answers is not in the document]
Comment (venkatr21) [reply sdk/loadtestservice/azure-resourcemanager-loadtestservice/pom.xml]: Got it, undo the pom changes

File: sdk/loadtestservice/azure-resourcemanager-loadtestservice/src/samples/java/com/azure/resourcemanager/loadtestservice/ReadmeSamples.java [hunk 2]
```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```
Comment (weidongxu-microsoft) [inline sdk/loadtestservice/azure-resourcemanager-loadtestservice/src/samples/java/com/azure/resourcemanager/loadtestservice/ReadmeSamples.java:3]: nit, remove this line as it is not generated.

Comment (weidongxu-microsoft) [reply sdk/loadtestservice/azure-resourcemanager-loadtestservice/src/samples/java/com/azure/resourcemanager/loadtestservice/ReadmeSamples.java:3]: There is other style problem reported in this file, please check CI and fix.

Comment (weidongxu-microsoft) [reply sdk/loadtestservice/azure-resourcemanager-loadtestservice/src/samples/java/com/azure/resourcemanager/loadtestservice/ReadmeSamples.java]: you need to have the copyright (first 2 lines), but not the 3rd about "generated"

**Comment by venkatr21** [comment]
@weidongxu-microsoft If the PR looks good, can we merge it?

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**